repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/examples/container.cfg.xml

```diff
@@ -32,4 +32,10 @@
 -->
         </ssl-configuration>
     </deployment-config>
+
+    <cluster-config cluster-id="internal-cluster">
```

Comment:
This brings up an interesting issue, what happens when the container config updates before the system model when a node is being added? I think it'd be ok.